src/metrics.rs

```diff
@@ -43,6 +54,12 @@ pub(crate) enum InstrumentType {
     UpDownCounterF64(f64),
     HistogramU64(u64),
     HistogramF64(f64),
+    #[cfg(feature="otel_unstable")]
+    GaugeU64(u64),
+    #[cfg(feature="otel_unstable")]
+    GaugeF64(f64),
+    #[cfg(feature="otel_unstable")]
+    GaugeI64(i64),
```

Comment:
Nit: ordering above suggests that `i64` goes before `f64` -- there are no examples of ordering between `u64` and `i64` but it probably makes sense to keep it in that order.

Reply:
[fixed](https://github.com/tokio-rs/tracing-opentelemetry/pull/129/commits/25e7756839d2d61f47e1f736567cb09c4c2ce87e)